src/hooks/useEvent/useEvent.tsx

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useLayoutEffect, useRef } from 'react'
+
+type AnyFunction = (...args: any[]) => any
```

Comment:
This should live in either a common type:
- either in `src/types/function.ts`
- or possibly in a `src/types/common.ts` (I have a preference for this one, it's only the second common type so far so namespacing them like I did for the `object.ts` is probably overkill)

Reply:
Fixed ! I put it in `src/types/common.ts` as suggested